pkg/ansible/runner/runner_test.go

```diff
@@ -63,111 +51,88 @@ func checkCmdFunc(t *testing.T, cmdFunc cmdFuncType, role, playbook string, watc
 }
 
 func TestNew(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Unable to get working director: %v", err)
+	}
+	validPlaybook := filepath.Join(cwd, "testdata", "playbook.yml")
```

Comment:
Since we are now validating watches, the role and playbook paths have to be legit.